app/styles/tab.module.css

```diff
@@ -0,0 +1,19 @@
+.tab {
+  border-radius: 9px;
+  width: 11rem;
+  height: 3rem;
+  color: var(--color-persian-indigo);
```

Comment:
I don't think we need extra color variables. 
The theme is already defined by the set of colors. 
So may be we can use that colors/